tests/New-DbaAgentOperator.Tests.ps1

```diff
@@ -0,0 +1,26 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [array]$params = ([Management.Automation.CommandMetaData]$ExecutionContext.SessionState.InvokeCommand.GetCommand($CommandName, 'Function')).Parameters.Keys
+        [object[]]$knownParameters = 'SqlInstance', 'SqlCredential', 'OperatorName', 'EmailAddress', 'NetSendAddress', 'PagerAddress', 'PagerDay', 'SaturdayStartTime', 'SaturdayEndTime', 'SundayStartTime', 'SundayEndTime', 'WeekendStartTime', 'WeekendEndTime', 'IsFailsafeOperator', 'FailsafeNotificationMethod', 'Force', 'EnableException'
+        It "Should only contain our specific parameters" {
+            Compare-Object -ReferenceObject $knownParameters -DifferenceObject $params | Should -BeNullOrEmpty
+        }
+    }
+}
+
+Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
+    Context "New Agent Operator is added properly" {
+
+        It "Should have the right name" {
+            $results = New-DbaAgentOperator -SqlInstance $script:instance2 -Operator DBA -EmailAddress operator@operator.com -PagerDay Everyday
```

Comment:
Running this command after making corrections noted in the function and it errors about a `Name` property not existing. Not sure why we don't get this error in our logs for Pester though.